pycbc/events/stat.py

```diff
@@ -50,6 +49,8 @@ def __init__(self, files):
             f = h5py.File(filename, 'r')
             stat = f.attrs['stat']
             self.files[stat] = f
+        self.single_dtype = numpy.float32
```

Comment:
Please add a comment to explain what single_dtype is intended to do here or in general for Stat subclasses.